integrationTests/multiShard/hardFork/hardFork_test.go

```diff
@@ -0,0 +1,302 @@
+package hardFork
```

Comment:
make the branch to reference the other branch, which is still in review. This way the code to be reviewed is less.

Reply:
done.